sphinx/ext/napoleon/__init__.py

```diff
@@ -256,7 +256,9 @@ def __unicode__(self):
 
         If an entry is just a string, it is interpreted as a header for a generic
         section. If the entry is a tuple/list/indexed container, the first entry
-        is the name of the section, the second is the section key to emulate.
+        is the name of the section, the second is the section key to emulate. If the
+        second entry value is "display_like_parameters" or "display_like_returns", the
+        custom section will be displayed like the parameters section or returns section.
```

Comment:
In addition to this change, please update document too.

Reply:
Do you mean I should also update the description in [`napoleon/docstring.py`](sphinx/ext/napoleon/docstring.py)? If not, could you point me to the document file? Thank you!

Reply:
I meant doc/usage/extensions/napoleon.rst. It has been built as https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html.